cloud/k8s/deploy/kontain-deploy/ci/set_ci_env.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 Kontain
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kontain-node-initializer
+spec:
+  template:
+    spec:
+      containers:
+      - name: node-initializer
+        env:
+        - name: KONTAIN_RELEASE_URL
+          value: http://host.minikube.internal:8000/kontain_bin.tar.gz
```

Comment:
just curious how does **host.minikube.internal** get resolved.

Reply:
Minikube adds it to coredns automagically.